### PR TITLE
Back end updates for feedback api

### PR DESCRIPTION
**Pull request by djnunez-aot**
-Back end updates for feedback api

-Update to front end endpoints

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/src/met_api/models/feedback.py [hunk 1]
```diff
@@ -60,3 +65,27 @@ def create_feedback(feedback):
...
+        """Update feedback by ID."""
+        feedback = cls.query.get(feedback_id)
+        if not feedback:
+            return None  # Feedback not found
+
+        for key, value in feedback_data.items():
+            if hasattr(feedback, key):
+                setattr(feedback, key, value)
+
+        db.session.commit()
```
Comment (jadmsaadaot) [inline met-api/src/met_api/models/feedback.py:90]: I think we can use the BaseModel save method here. Just feedback.save()

File: met-api/src/met_api/resources/feedback.py [hunk 2]
```diff
@@ -52,7 +49,8 @@ def get():
                 sort_key=args.get('sort_key', 'name', str),
                 sort_order=args.get('sort_order', 'asc', str),
             )
-            feedback_records = FeedbackService().get_feedback_paginated(pagination_options, search_text)
+            feedback_records = FeedbackService().get_feedback_paginated(
+                pagination_options, search_text)
```
Comment (jadmsaadaot) [inline met-api/src/met_api/resources/feedback.py:53]: is there a reason for this?

File: met-api/src/met_api/resources/feedback.py [hunk 3]
```diff
@@ -66,7 +64,8 @@ def post():
         try:
             user_id = TokenInfo.get_id()
             request_json = request.get_json()
-            valid_format, errors = schema_utils.validate(request_json, 'feedback')
+            valid_format, errors = schema_utils.validate(
```
Comment (jadmsaadaot) [inline met-api/src/met_api/resources/feedback.py:67]: is there a reason for this

**Review by jadmsaadaot — APPROVED** [review]

**Review by saravanpa-aot — COMMENTED** [review]
good job
can you add some tests as well plz

**Pull request merged**